carp.data.core/src/commonMain/kotlin/dk/cachet/carp/data/application/Measurement.kt

```diff
@@ -40,6 +40,17 @@ data class Measurement<out TData : Data>(
     fun getDataTimeType(): DataTimeType =
         if ( sensorEndTime == null ) DataTimeType.POINT
         else DataTimeType.TIME_SPAN
+
+    /**
+     * Convert this [Measurement] to one synchronized to UTC time using [syncPoint].
+     */
+    fun synchronizeToUTC( syncPoint: SyncPoint ): Measurement<TData> =
+        copy(
+            sensorStartTime = syncPoint.synchronizeTimestamp( sensorStartTime ),
```

Comment:
`syncPoint.synchronizeTimestamp` returns `syncedTimestamp` value based on `syncPoint` and `sensorStartTime` value. I doubt does this function really convert `sensorStartTime` and `sensorEndTime` to UTC value?

Reply:
Yes. The `synchronizeTimestamp` function returns `syncedTimestamp`. This corresponds to microseconds since the Unix epoch, which is expressed as a moment in UTC time, and thus more or less stores UTC time indirectly
But I can see how the shorthand in the documentation did more harm then good. :) I will update.